### PR TITLE
[javascript] Add CVE-2017-16175

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@sara-02 PTAL, thanks :)

File: database/javascript/2017/16175.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+affected:
+    - name: ewgaddis.lab6
+      version:
+        - "<unable-to-determine>"
+
+
+# Additional information:
+# Package name candidates:
+# 11.344348 ewgaddis.lab6
```
Comment (sara-02) [inline database/javascript/2017/16175.yaml]: This is the correct package name `ewgaddis.lab6` as per description.

File: database/javascript/2017/16175.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+description: >
+    ewgaddis.lab6 is a file server. ewgaddis.lab6 is vulnerable to a directory traversal issue, giving an attacker access to the filesystem by placing "../" in the url.
+cvss_v2: 5.0
+references:
+    - https://github.com/JacksonGL/NPM-Vuln-PoC/blob/master/directory-traversal/ewgaddis.lab6
+    - https://nodesecurity.io/advisories/465
+
+affected:
+    - name: ewgaddis.lab6
+      version:
```
Comment (sara-02) [inline database/javascript/2017/16175.yaml]: https://nvd.nist.gov/vuln/detail/CVE-2017-16175?cpeVersion=2.2#vulnConfigurationsArea

Comment (msrb) [reply database/javascript/2017/16175.yaml]: +1 please update the PR

**Pull request merged**